ESSArch_PP/frontend/static/frontend/views/preview_appraisal_modal.html

```diff
@@ -32,3 +32,8 @@ <h4>{{'FILES_TO_BE_REMOVED' | translate}}</h4>
         <button class="btn btn-primary" type="button" ng-click="$ctrl.ok()" focused="true">{{'RUN' | translate}}</button>
         <button class="btn btn-warning" type="button" ng-click="$ctrl.cancel()">{{'CANCEL' | translate}}</button>
     </div>
+</div>
```

Comment:
This `</div>` doesn't have a matching `<div>`, fix the indentation of the file instead.